README.md

```diff
@@ -1,10 +1,20 @@
-[![Build Status](https://travis-ci.org/dedis/cosi.svg?branch=master)](https://travis-ci.org/dedis/cosi)
-[![Coverage Status](https://coveralls.io/repos/github/dedis/cosi/badge.svg?branch=master)](https://coveralls.io/github/dedis/cosi?branch=master)
+[![Build Status](https://travis-ci.org/dedis/cothority.svg?branch=master)](https://travis-ci.org/dedis/cothority)
+[![Coverage Status](https://coveralls.io/repos/github/dedis/cothority/badge.svg?branch=master)](https://coveralls.io/github/dedis/costhority?branch=master)
 
 
 # CoSi
 
-This repository implements the CoSi protocol for scalable collective signing.
+**Update: Development of CoSi moved to the 
+[cothority](https://github.com/dedis/cothority/) repository. 
+For an updated version of this document see: 
+https://github.com/dedis/cothority/blob/master/app/cosi
+This repository only contains the first unstable 
+[`v0` branch](https://gopkg.in/dedis/cosi.v0) and some outdated 
```

Comment:
"Version zero (v0) is reserved for packages that are so immature that offering any kind of API stability guarantees would be unreasonable. This is equivalent to labeling the package as alpha or beta quality, and as such the use of these packages as dependencies of stable packages and applications is discouraged."
http://labix.org/gopkg.in